autoPlay.user.js

```diff
@@ -1155,12 +1155,16 @@ function useGoldRainIfRelevant() {
 			return;
 		}
 
+		if (Math.random() > s().CurrentScene().m_rgGameData.level / 10000) {
```

Comment:
Gold Rain is still possible but less likely at lower levels.  By level 5000 there is a 50% chance of triggering gold rain each time the script runs (so if the boss takes several seconds to kill it is quite likely that it will be triggered).

With multiple people running the script, gold farming can still happen at lower levels but is weighted to later levels where it will be more valuable.